### DISCUSSION TITLE
已经安装完如何同步和升级？

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#92] [Q&A] [answered]
如题，没有升级和安装指南啊。
求大佬指点

**Comment by Kutius** [comment, marked as answer]
github同步上游就行了

**Reply by tkfanswind** [reply]
> github同步上游就行了

咋操作？
merge？

**Reply by Kutius** [reply]
https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
